addons/calendar/tests/test_calendar.py

```diff
@@ -381,6 +381,30 @@ def _test_emails_has_attachment(self, partners):
         # no more email should be sent
         _test_one_mail_per_attendee(self, partners)
 
+        # Check for attachments when public user creates an event ( simulates scheduling appointment from an incognito window)
+        attachment_2 = self.env['ir.attachment'].create({
+            'datas': base64.b64encode(bytes("Event Attachment 2", 'utf-8')),
+            'name': 'fileText_attachment_2.txt',
+            'mimetype': 'text/plain'
+        })
+        self.env.ref('calendar.calendar_template_meeting_invitation').attachment_ids += attachment_2
+
+        self.partner_staff = self.env['res.partner'].create({
+            'name': 'partner_staff',
+            'email': 'partner_staff@gmail.com',
```

Comment:
Don't use real emails (use @example.com or stuff like that), better safe than sorry :)